spring-integration-core/src/main/java/org/springframework/integration/channel/FluxMessageChannel.java

```diff
@@ -52,11 +52,11 @@
 
 	private volatile boolean upstreamSubscribed;
 
-	public ReactiveChannel() {
+	public FluxMessageChannel() {
 		this(DirectProcessor.create());
 	}
 
-	public ReactiveChannel(FluxProcessor<Message<?>, Message<?>> processor) {
+	public FluxMessageChannel(FluxProcessor<Message<?>, Message<?>> processor) {
```

Comment:
That's still isn't clear.
I talk about `Processor` ,but you mention `Mono` somehow...
Sorry, I don't follow